lib/attr.h

```diff
@@ -5,7 +5,7 @@
 #include "config.h"
 
 
-#if defined(__GNUC__)
+#if defined(__GNUC__) && (__GNUC__ >= 10)
```

Comment:
Hmm, we can simplify by removing the defined() clause.  Undefined macros have the value 0 in `#if` directives:

```c
alx@devuan:~/tmp/gcc$ cat cpp.c
#if (ASD > 1)
#warning TRUE
#else
#warning FALSE
#endif

#if (ASD < 1)
#warning TRUE
#else
#warning FALSE
#endif

#if (ASD == 0)
#warning TRUE
#else
#warning FALSE
#endif
alx@devuan:~/tmp/gcc$ cc -Wall -Wextra -S cpp.c 
cpp.c:4:2: warning: #warning FALSE [-Wcpp]
    4 | #warning FALSE
      |  ^~~~~~~
cpp.c:8:2: warning: #warning TRUE [-Wcpp]
    8 | #warning TRUE
      |  ^~~~~~~
cpp.c:14:2: warning: #warning TRUE [-Wcpp]
   14 | #warning TRUE
      |  ^~~~~~~
```

Reply:
Done